src/Hyperion/Extensions/TypeEx.cs

```diff
@@ -123,6 +123,11 @@ private static Type GetTypeFromManifestName(Stream stream, DeserializerSession s
                 {
                     shortName = shortName.Replace("System.Private.CoreLib,%core%", "mscorlib,%core%");
                 }
+                
+                if (shortName.Contains("System.Drawing.Primitives"))
```

Comment:
Ah, this is another case of where a single namespace was changed between .NET framework  and .NET core?

Reply:
They decided that structs are primitives and moved all structs inside `System.Drawing` into `System.Drawing.Primitives`